clash-prelude/src/Clash/Explicit/BlockRam.hs

```diff
@@ -914,7 +926,7 @@ blockRamU#
   -> Signal dom a
   -- ^ Value to write (at address @w@)
   -> Signal dom a
-  -- ^ Value of the @blockRAM@ at address @r@ from the previous clock cycle
+  -- ^ Value of the BRAMblockRAM@ at address @r@ from the previous clock cycle
```

Comment:
> BRAMblockRAM

_Is a normal block ram not blocky enough? Try new block RAM block RAM today_

Reply:
_BRAM BRAM BRAAAAAM_